src/codewhisperer/vue/root.vue

```diff
@@ -107,9 +107,7 @@
                                 you provide to the Beta Service, and that multiple users may receive the same or similar
                                 output. For clarity, although these computational results are Your Content, the same or
                                 similar outputs are independently created, and AWS will not be restricted from providing
-                                the same or similar output to other customers. In addition, the CodeWhisperer Beta
-                                Service may produce computational results that are attributed to AWS or our affiliates,
-                                and we retain the underlying intellectual property rights in that Content;
+                                the same or similar output to other customers.
```

Comment:
I think some other unrelated changes might have got pulled into this PR (a few codewhisperer files)

Reply:
I think it's just a Github bug because the relevant commit https://github.com/aws/aws-toolkit-vscode/commit/6f979456eccf8c0be81616f80ffd602bcfb6293b is in the target branch. Maybe if there's a merge conflict it shows a different diff?

It should go away once the merge conflict is resolved